strawberryfields/engine.py

```diff
@@ -540,17 +548,19 @@ def run(self, program: Program, *, compile_options=None, **kwargs) -> Optional[R
             ``None`` otherwise
         """
         job = self.run_async(program, compile_options=compile_options, **kwargs)
+        log.info("Submitted job {}".format(job.id))
         try:
             while True:
                 job.refresh()
                 if job.status == "complete":
                     return job.result
                 if job.status == "failed":
                     log.warning(
-                        "The remote job failed due to an internal server error; "
-                        "please try again."
+                        "The remote job {} failed due to an internal server error; "
+                        "please try again.".format(job.id)
```

Comment:
:+1:

Reply:
What was pylint complaining about out of curiosity?

Reply:
`logging-format-interpolation` (similarly to [this thread](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation))